src/ts/client/index.ts

```diff
@@ -99,5 +99,7 @@ export async function start(options = {}): Promise<void> {
       document.querySelector('#gjs').classList.remove('silex-dialog-hide')
       config.emit(ClientEvent.STARTUP_END, { editor, config })
     }, 100)
+    const um = editor.UndoManager;
```

Comment:
@oliviermgx would you please do another PR with these `;` removed?
I always omit them and the linter fails and breaks the build:
https://github.com/silexlabs/Silex/actions/runs/8200630916/job/22427762798